pkg/ibmvpc/examples/multiple_vpcs_all_vpcssubnetsBased_withPGW.txt

```diff
@@ -0,0 +1,98 @@
+ky-testenv-private-subnet-2 => ky-testenv-transit-subnet-2 : protocol: TCP dst-ports: 443
+ky-testenv-private-subnet-2 => ky-testenv-transit-subnet-3 : protocol: TCP dst-ports: 443
+ky-testenv-private-subnet-3 => ky-testenv-edge-subnet-1 : protocol: TCP dst-ports: 443
+ky-testenv-private-subnet-3 => ky-testenv-edge-subnet-2 : protocol: TCP dst-ports: 443
+ky-testenv-private-subnet-3 => ky-testenv-edge-subnet-3 : protocol: TCP dst-ports: 443
+ky-testenv-private-subnet-3 => ky-testenv-private-subnet-1 : protocol: TCP src-ports: 443 dst-ports: 443
+ky-testenv-private-subnet-3 => ky-testenv-private-subnet-2 : protocol: TCP src-ports: 443 dst-ports: 443
+ky-testenv-private-subnet-3 => ky-testenv-transit-subnet-1 : protocol: TCP dst-ports: 443
+ky-testenv-private-subnet-3 => ky-testenv-transit-subnet-2 : protocol: TCP dst-ports: 443
+ky-testenv-private-subnet-3 => ky-testenv-transit-subnet-3 : protocol: TCP dst-ports: 443
+ky-testenv-transit-subnet-1 => ky-testenv-edge-subnet-1 : All Connections
+ky-testenv-transit-subnet-1 => ky-testenv-edge-subnet-2 : All Connections
+ky-testenv-transit-subnet-1 => ky-testenv-edge-subnet-3 : All Connections
+ky-testenv-transit-subnet-1 => ky-testenv-private-subnet-1 : protocol: TCP src-ports: 443
+ky-testenv-transit-subnet-1 => ky-testenv-private-subnet-2 : protocol: TCP src-ports: 443
+ky-testenv-transit-subnet-1 => ky-testenv-private-subnet-3 : protocol: TCP src-ports: 443
+ky-testenv-transit-subnet-1 => ky-testenv-transit-subnet-2 : All Connections
+ky-testenv-transit-subnet-1 => ky-testenv-transit-subnet-3 : All Connections
+ky-testenv-transit-subnet-2 => ky-testenv-edge-subnet-1 : All Connections
+ky-testenv-transit-subnet-2 => ky-testenv-edge-subnet-2 : All Connections
+ky-testenv-transit-subnet-2 => ky-testenv-edge-subnet-3 : All Connections
+ky-testenv-transit-subnet-2 => ky-testenv-private-subnet-1 : protocol: TCP src-ports: 443
+ky-testenv-transit-subnet-2 => ky-testenv-private-subnet-2 : protocol: TCP src-ports: 443
+ky-testenv-transit-subnet-2 => ky-testenv-private-subnet-3 : protocol: TCP src-ports: 443
+ky-testenv-transit-subnet-2 => ky-testenv-transit-subnet-1 : All Connections
+ky-testenv-transit-subnet-2 => ky-testenv-transit-subnet-3 : All Connections
+ky-testenv-transit-subnet-3 => ky-testenv-edge-subnet-1 : All Connections
+ky-testenv-transit-subnet-3 => ky-testenv-edge-subnet-2 : All Connections
+ky-testenv-transit-subnet-3 => ky-testenv-edge-subnet-3 : All Connections
+ky-testenv-transit-subnet-3 => ky-testenv-private-subnet-1 : protocol: TCP src-ports: 443
+ky-testenv-transit-subnet-3 => ky-testenv-private-subnet-2 : protocol: TCP src-ports: 443
+ky-testenv-transit-subnet-3 => ky-testenv-private-subnet-3 : protocol: TCP src-ports: 443
+ky-testenv-transit-subnet-3 => ky-testenv-transit-subnet-1 : All Connections
+ky-testenv-transit-subnet-3 => ky-testenv-transit-subnet-2 : All Connections
+
+connections are stateful unless marked with *
```

Comment:
Would be nice if we can have this comment only once per report rather than after every VPC report.

Reply:
Will be handled in a related open issue #129